gramps/plugins/export/exportgedcom.py

```diff
@@ -513,14 +513,16 @@ def _gender(self, person):
         Write out the gender of the person to the file.
 
         If the gender is not male or female, simply do not output anything.
-        The only valid values are M (male) or F (female). So if the geneder is
-        unknown, we output nothing.
+        The only valid values are M (male), F (female) and X (other). So if
```

Comment:
X is only valid for Gedcom >= 7 so should this be handled differently here?

Reply:
Possibly.  I'm not sure which version of Gedcom we want to support in the export. We could just export "X" as "U".

Reply:
I think that might make sense for 5.5.1 and attach an auto generated FACT SEX attribute with the Other setting so at least the data doesn't get lost?